specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Migrations.json

```diff
@@ -0,0 +1,822 @@
+          "x-ms-external": true,
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
+        },
+        "targetServerPassword": {
+          "type": "string",
+          "x-ms-secret": true,
+          "x-ms-external": true,
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
+        }
+      },
+      "description": "Server admin credentials."
+    },
+    "DBServerMetadata": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        },
+        "storageMb": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "The SKU (pricing tier) of the server."
+        }
+      },
+      "description": "Database server metadata."
+    },
+    "MigrationDetailsLevel": {
+      "enum": [
+        "Default",
+        "Summary",
+        "Full"
+      ],
+      "type": "string",
+      "description": "Migration details level.",
+      "x-ms-enum": {
+        "name": "MigrationDetailsLevel",
+        "modelAsString": true
+      }
+    },
+    "MigrationState": {
+      "enum": [
+        "InProgress",
+        "WaitingForUserAction",
+        "Canceled",
+        "Failed",
+        "Succeeded"
+      ],
+      "type": "string",
+      "description": "Migration state.",
+      "x-ms-enum": {
+        "name": "MigrationState",
+        "modelAsString": true
+      }
+    },
+    "MigrationStatus": {
+      "type": "object",
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/MigrationState",
+          "readOnly": true
+        },
+        "error": {
+          "type": "string",
+          "readOnly": true
+        },
+        "currentSubStateDetails": {
+          "$ref": "#/definitions/MigrationSubStateDetails",
+          "readOnly": true
+        }
+      },
+      "description": "Migration status."
+    },
+    "MigrationSubState": {
+      "enum": [
+        "PerformingPreRequisiteSteps",
+        "WaitingForLogicalReplicationSetupRequestOnSourceDB",
+        "WaitingForDBsToMigrateSpecification",
+        "WaitingForTargetDBOverwriteConfirmation",
+        "WaitingForDataMigrationScheduling",
+        "WaitingForDataMigrationWindow",
+        "MigratingData",
+        "WaitingForCutoverTrigger",
+        "CompletingMigration",
+        "Completed"
+      ],
+      "type": "string",
+      "description": "Migration sub state.",
+      "x-ms-enum": {
+        "name": "MigrationSubState",
+        "modelAsString": true
+      }
+    },
+    "MigrationSubStateDetails": {
+      "type": "object",
+      "properties": {
+        "currentSubState": {
+          "$ref": "#/definitions/MigrationSubState",
+          "readOnly": true
+        }
+      },
+      "description": "Migration sub state details."
+    },
+    "Sku": {
```

Comment:
Define this exactly the same as the Sku in other swagger files in this directory
Or rename this to something else if they are really different

Reply:
The other Sku is defined here: https://github.com/Azure/azure-rest-api-specs/blob/65467c71b96b9fae47ed74d9866a3494030336d4/specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/FlexibleServers.json#L679
These are almost the same.
If you do not have explicit requirement on duplicating them, here is a choice you could choose from:
1. Remove this definition here and change everywhere referencing this definition to a ref that reference the `Sku` defined in the other file.
2. Copy the definition in the other file exactly here, to ensure the definition here is exactly the same as the one in the other file.

Reply:
Tried all these, I think that issue is not becoz of my changes